src/languages/en.ts

```diff
@@ -4240,6 +4242,19 @@ const translations = {
                 moreDetails: 'for more details.',
                 gotIt: 'Got it, thanks',
             },
+            commonFeatures: {
+                title: 'Upgrade Workspace to Control',
+                note: 'Get access to all our most advanced functionality, including:',
+                benefits: {
+                    note: 'The Control plan starts at $9 per active member per month.',
+                    learnMore: 'learn more',
+                    pricing: 'about our plans and pricing.',
+                    benefit1: 'Advanced accounting connections (NetSuite, Sage Intacct and more)',
+                    benefit2: 'Expense rules',
+                    benefit3: 'Multiple approval workflows',
+                    benefit4: 'Enhanced security controls',
+                },
+            },
```

Comment:
These translations are yet to be confirmed here: https://expensify.slack.com/archives/C01GTK53T8Q/p1731048615013379?thread_ts=1730105986.836259&cid=C01GTK53T8Q

Reply:
@carlosmiceli Before merging, I wanted to let you know that these translations are not yet confirmed.

Reply:
Great, let's keep these comments open so we can go back to them to see what we have yet to change/merge.